engines/query-sparql-file/package.json

```diff
@@ -204,9 +202,5 @@
     "@comunica/runner-cli": "^3.2.1",
     "@comunica/types": "^3.2.1",
     "process": "^0.11.10"
-  },
-  "browser": {
```

Comment:
We also need to keep this.

Reply:
(see the comment above on the `browser` build script)